packages/react-components/react-drawer/src/components/Drawer/useDrawerStyles.styles.ts

```diff
@@ -12,24 +12,39 @@ export const drawerClassNames: SlotClassNames<DrawerSlots> = {
  */
 export const drawerCSSVars = {
   size: '--fui-Drawer--size',
+  borderRadius: '--fui-Drawer--borderRadius',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
```

Comment:
Drawer will not have its own classNames to target in this case. Users will need to know that to target Drawer classNames, they will need to import Dialog classNames. Is that the intention?

Reply:
In the case of `inline` Drawer, there's not even a className that's targetable

Reply:
What do you mean about "classNames to target"? This kind of className: "fui-Drawer"?